pkgs/development/python-modules/pandas/default.nix

```diff
@@ -8,37 +9,43 @@
 , cython
 , dateutil
 , html5lib
+, jinja2
 , lxml
 , numexpr
 , openpyxl
 , pytz
-, sqlalchemy
 , scipy
+, sqlalchemy
 , tables
+, XlsxWriter
 , xlrd
 , xlwt
-# Test Inputs
+# Test inputs
 , glibcLocales
 , hypothesis
 , pytestCheckHook
+, pytest-xdist
+, pytest-asyncio
 # Darwin inputs
 , runtimeShell
-, libcxx ? null
+, libcxx
 }:
 
 buildPythonPackage rec {
   pname = "pandas";
-  version = "1.1.5";
+  version = "1.2.2";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "06vhk75hmzgv1sfbjzgnsw9x10h7y6bd6s6z7d6lfnn7wcgc83zi";
+    sha256 = "sha256-FO2EtGPpuEyP+TCKebBL9ZGuMSKjdu4PYsaKG9kXp3M=";
   };
   # See https://github.com/scipy/scipy/issues/13585 and https://github.com/pandas-dev/pandas/pull/40020
   patches = [ ./fix-tests.patch ];
```

Comment:
This patch can then be dropped, since the fix was backported to 1.2.x.

https://github.com/pandas-dev/pandas/commit/b14ffd8e17cabf445fb7709d6518639e9791a1bf

Reply:
Patch removed